File: easy/cheer-creator/CheerCreator.java
import java.util.Scanner;

public final class CheerCreator {
    public static void main(final String[] args) {
        final var yards = new Scanner(System.in).nextInt();
        
        if (yards > 10) {
            System.out.print("High Five");
        } else if (yards < 1) {
            System.out.print("shh");
        } else {
            System.out.print("Ra!".repeat(yards));
        }
    }
}

File: easy/cheer-creator/cheer_creator.cpp
#include <iostream>
using namespace std;

int main() {
    int yards;
    cin >> yards;
    
    if (yards > 10) cout << "High Five";
    else if (yards < 1) cout << "shh";
    else while (yards--) cout << "Ra!";
    
    return 0;
}
